pydomcfg/tests/test_zco.py

```diff
@@ -0,0 +1,132 @@
+"""
+Test for zco
+"""
+
+import numpy as np
+import xarray as xr
+
+from pydomcfg.domzgr.zco import Zco
+from pydomcfg.tests.bathymetry import Bathymetry
+
+
+def test_zco():
```

Comment:
You could use the parametrize decorator: https://docs.pytest.org/en/6.2.x/parametrize.html
When we'll have many tests, the pytest log will be much easier to intrepret.
```python
import pytest

@pytest.mark.parametrize("name", ["ORCA2", "uniform"])
@pytest.mark.parametrize("ppkth", [21.43336197938, 0.0])
@pytest.mark.parametrize("ppacr", [3, 0.0])
...
def test_zco(name, ppkth, ppacr, ...):
```